### DISCUSSION TITLE
Is there a way to add virtual attribute to model?

### DISCUSSION THREAD

**Discussion by darron1217** [strapi/strapi#8152] [Legacy Topics]
In many case, we need virtual attribute which combines or mutates original attributes.

Case 1. `fullName` which combines `lastName` and `firstName`
Case 2. `studentCount` which aggregates count from relation `students`

I want to achieve it on GraphQL. Is there a proper way to achieve it?

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

**Reply by darron1217** [reply]
Got it
